ProteoformSuiteInternal/Results.cs

```diff
@@ -0,0 +1,434 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Yes I'm not realizing if I didn't go into the folder and mnaually delete the file it stayed, even if it was deleted in upstream merges.

I think there's lot of remaining code from that throughout Internal/GUI that I'll go through and clean up on the main branch.